fluentlenium-core/src/main/java/org/fluentlenium/core/FluentAdapter.java

```diff
@@ -98,28 +107,38 @@ private void injectPageIntoContainer(Fluent container)
         return container;
     }
 
-
     protected <T extends FluentPage> T initClass(Class<T> cls, Object... params) {
+        T page = null;
```

Comment:
What happened if the page is not set? Are you returning null object and NPE will be thrown on related methods?